### PR TITLE
display fake contacts on a list

### PR DESCRIPTION
**Pull request by arreche**

### CODE REVIEW

File: src/containers/main/main.js [hunk 1]
```diff
@@ -1,9 +1,36 @@
 import  React, { Component } from 'react';
 import Toolbar from '../../components/toolbar/toolbar';
+import Contacts from '../../components/contacts/contacts';
+
+const contacts = [
+   {
+      "name":{
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)

File: src/containers/main/main.js [hunk 2]
```diff
@@ -1,9 +1,36 @@
...
+         "first":"manuela"
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)
* Error - Strings must use singlequote. (quotes)

File: src/containers/main/main.js [hunk 3]
```diff
@@ -1,9 +1,36 @@
...
+      },
+      "picture":{
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)

File: src/containers/main/main.js [hunk 4]
```diff
@@ -1,9 +1,36 @@
...
+         "thumbnail":"http://api.randomuser.me/portraits/thumb/women/39.jpg"
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)
* Error - Strings must use singlequote. (quotes)

File: src/containers/main/main.js [hunk 5]
```diff
@@ -1,9 +1,36 @@
...
+      }
+   },
+   {
+      "name":{
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)

File: src/containers/main/main.js [hunk 6]
```diff
@@ -1,9 +1,36 @@
...
+         "first":"barbara"
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)
* Error - Strings must use singlequote. (quotes)

File: src/containers/main/main.js [hunk 7]
```diff
@@ -1,9 +1,36 @@
...
+      },
+      "picture":{
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)

File: src/containers/main/main.js [hunk 8]
```diff
@@ -1,9 +1,36 @@
...
+         "thumbnail":"http://api.randomuser.me/portraits/thumb/women/74.jpg"
```
Comment (arreche) [inline src/containers/main/main.js]: __[eslint]__
* Error - Strings must use singlequote. (quotes)
* Error - Strings must use singlequote. (quotes)

**Comment by yeraycaballero** [comment]
Would you change those double quotes to single quotes?. go ahead and merge it

**Pull request merged**